f2/src/utils/encryptedEmail.js

```diff
@@ -110,10 +136,11 @@ const getAllCerts = uidList => {
   else console.warn('Encrypted Mail: No certs to fetch!')
 }
 
-const encryptAndSend = async (uidList, message) => {
-  if (uidList)
+const encryptAndSend = async (uidList, data, message) => {
```

Comment:
🙂 I know you didn’t commit your server.js code because you had hacked it up, but we’ll have to add “data” to the params of encryptAndSend when we call it.